src/ripple/ledger/impl/PaymentSandbox.cpp

```diff
@@ -260,4 +262,138 @@ PaymentSandbox::apply (PaymentSandbox& to)
     tab_.apply(to.tab_);
 }
 
+std::pair<std::map<std::tuple<AccountID, AccountID, Currency>, STAmount>, XRPAmount>
+PaymentSandbox::balancesChanges (ReadView const& view) const
+{
+    using key = std::tuple<AccountID, AccountID, Currency>;
+    // Map of delta trust lines
+    // As a special case, when both ends of the trust line are the same currency, then
+    // it's delta currency for that issuer
+    // To get the change in XRP balance, Account == root, issuer == root, currency == XRP
+    std::map<key, STAmount> result;
+
+    // populate a dictionary with low/high/currency/delta // this can be
+    // compared with the old flow, and xrp deltas// can be checked with amount
+    // destroyed
```

Comment:
The "//" occurrences in this paragraph look like there may be a comment wrapping peculiarity.  Are they intentional?